app/lib/check_cached_fields.rb

```diff
@@ -97,12 +97,15 @@ def index_cached_field(options, value, name, obj)
           update_pg: options[:update_pg],
           pg_field_name: options[:pg_field_name],
         }
-        self.delay_for(1.second).index_cached_field_bg(index_options, value, name, obj)
+        obj_data = { klass: obj.class.name, id: obj.id }
+        self.delay_for(1.second).index_cached_field_bg(index_options, value, name, YAML::dump(obj_data))
```

Comment:
Question: Why pass an argument `obj_data` instead of two arguments `klass` and `id`? Is it just to avoid that current jobs fail? If so, I agree it's a good idea! Just want to confirm the intent. And if so, I think you should add a comment to the code to explain the rationale.

Reply:
The current jobs will fail event I pass an argument `obj_data`  but I use `object_data`  to be consistent with other implementation.
I agreed with your suggestion to pass id and klass it'll be more clear.

Reply:
Done